projects/dotcms-field-elements/src/components/dot-multi-select/dot-multi-select.e2e.ts

```diff
@@ -1,100 +1,337 @@
+                expect(await optionElements[1].getProperty('selected')).toBe(false);
+                expect(await optionElements[2].getProperty('selected')).toBe(false);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('options', 'a|1,b|2,c|3');
+                element.setProperty('value', wrongValue);
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(await optionElements[0].getProperty('selected')).toBe(false);
+                expect(await optionElements[1].getProperty('selected')).toBe(false);
+                expect(await optionElements[2].getProperty('selected')).toBe(false);
+            });
+        });
     });
 
-    describe('Events', () => {
+    describe('@Events', () => {
         beforeEach(async () => {
+            page = await newE2EPage();
+            await page.setContent(`
+            <dot-multi-select
+                name="testName"
+                label="testLabel"
+                hint="testHint"
+                options="|,valueA|1,valueB|2"
+                value="2"
+                required-message="testErrorMsg"
+                required="true">
+            </dot-multi-select>`);
             spyStatusChangeEvent = await page.spyOnEvent('statusChange');
             spyValueChangeEvent = await page.spyOnEvent('valueChange');
+
+            element = await page.find('dot-multi-select');
         });
 
-        it('should emit "statusChange" & "valueChange"', async () => {
-            await page.select('select', '1', '2');
-            expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
-                name: 'testName',
-                status: {
-                    dotPristine: false,
-                    dotTouched: true,
-                    dotValid: true
-                }
+        describe('status and value change', () => {
+            it('should emit when option selected', async () => {
+                await page.select('select', '1');
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                    name: 'testName',
+                    status: {
+                        dotPristine: false,
+                        dotTouched: true,
+                        dotValid: true
+                    }
+                });
+                expect(spyValueChangeEvent).toHaveReceivedEventDetail({
+                    name: 'testName',
+                    value: '1'
+                });
             });
-            expect(spyValueChangeEvent).toHaveReceivedEventDetail({
-                name: 'testName',
-                value: '1,2'
+
+            it('should emit when 2 options selected', async () => {
+                await page.select('select', '1', '2');
+                expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
```

Comment:
Is this expect not the same as https://github.com/dotCMS/core-web/pull/1034/files#diff-614e68f946feb689b1b0fe5327e8c518R290 this one?